client/components/relatedWidget/RelatedCarousel.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import ScrollMenu from 'react-horizontal-scrolling-menu';
+import ProductList from './ProductList.jsx';
+import ArrowButton from './ArrowButton.jsx';
+import axios from 'axios';
+
+const selected = null;
+const ArrowLeft = ArrowButton({
+  text: '<',
+  className: 'arrow-prev'
+});
+const ArrowRight = ArrowButton({
+  text: '>',
+  className: 'arrow-next'
+});
+
+class RelatedCarousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: selected,
+      relatedIds: [],
+      relatedProducts: [],
+      relatedImages: []
+    };
+    this.productItems = ProductList(
```

Comment:
Interesting, are you basically adding an array of React component as a property on this class constructor?
So this.productItems would look something like: `[ <ProuctListItem />, <ProuctListItem />, <ProuctListItem /> ]`